.github/workflows/vulnerability-scanner-generate-database.yml

```diff
@@ -0,0 +1,99 @@
+      - "src/shared_modules/indexer_connector/**"
+      - "src/wazuh_modules/vulnerability_scanner/**"
+      - "src/wazuh_modules/http-request/**"
+
+jobs:
+  vulnerability-scanner-updaload-database:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+        with:
+          submodules: recursive
+
+      - name: Project dependencies
+        uses: ./.github/actions/vulnerability_scanner_deps
+
+      ########################
+      # Compilation          #
+      ########################
+
+      # Router
+      - name: Router - Compilation and test
+        uses: ./.github/actions/compile_and_test
+        with:
+          path: src/shared_modules/router
+
+      # Indexer connector
+      - name: Indexer connector - Compilation and test
+        uses: ./.github/actions/compile_and_test
+        with:
+          path: src/shared_modules/indexer_connector
+
+      # Content manager
+      - name: Content manager - Compilation and test
+        uses: ./.github/actions/compile_and_test
+        with:
+          path: src/shared_modules/content_manager
+
+      # Vulnerability scanner
+      - name: Vulnerability scanner - Compilation and test
+        uses: ./.github/actions/compile_and_test
+        with:
+          path: src/wazuh_modules/vulnerability_scanner
+
+# Only for test. Uncomment for real data
+#      - name: Vulnerability scanner - Run vd_scanner_testtool
+#        run: |
+#          ./src/wazuh_modules/vulnerability_scanner/build/testtool/scanner/vd_scanner_testtool -c src/wazuh_modules/vulnerability_scanner/testtool/scanner/config.json -t src/wazuh_modules/vulnerability_scanner/indexer/template/legacy-template.json -d
+#        shell: bash
+
+# Only for test. Uncomment for real data. Move to the next step, at the beginning
+# and remove dummy folder commands
+#  rm -rf queue/indexer
+#  rm -rf queue/sockets
+#  rm -rf queue/router
+
+      - name: Vulnerability scanner - Compress queue folder
+        run: |
+          mkdir -p queue/vd
+          mkdir -p queue/vd-updater
+          touch queue/vd/file1
+          touch queue/vd-updater/file1
+          echo "content vd" >> queue/vd/file1
+          echo "content updater" >> queue/vd-updater/file1
+          
+          VD_FILENAME=vd_1.0.0_vd_4.8.0.tar.xz
+          
+          echo "Compressing file"
+          tar -cJf ${VD_FILENAME} queue
+          
+          if ! [ -f ${VD_FILENAME} ]; then
+            echo "Error generating ${VD_FILENAME} file"
+          fi
+        shell: bash
+
+      - name: Vulnerability scanner - Uploading file to S3
+        uses: shallwefootball/s3-upload-action@master
+        with:
+          aws_key_id: ${{ secrets.FEED_AWS_KEY_ID }}
+          aws_secret_access_key: ${{ secrets.FEED_AWS_SECRET_ACCESS_KEY}}
+          aws_bucket: ${{ secrets.FEED_AWS_BUCKET }}
+          source_dir: './vd_1.0.0_vd_4.8.0.tar.xz'
+          destination_dir: 'deps/vulnerability_model_database/'
```

Comment:
Please, add a new line at the end of the file.

Reply:
Done